.github/workflows/main.yml

```diff
@@ -22,5 +22,5 @@ jobs:
         run: nix shell .#packages.x86_64-linux.reuse -c reuse lint
 
       - name: build nixfmt
-        run: nix build -L .#nixfmt-static
+        run: nix build -L .
```

Comment:
Is this really the correct command when your precious PR was about using stable nix? Iirc the nix command is not stable either

Reply:
Ah good point, I originally avoided this because of the `nix shell` above because there's no equivalent stable command, but really, it's just `nix-build .. && reuse lint`, changed it now :)